src/main/webapp/scheduled-interviews.jsp

```diff
@@ -23,6 +23,16 @@
             <li class="list-group-item">Interviewee: ${scheduledInterview.getInterviewee()}</li>
             <li class="list-group-item">Interviewer: ${scheduledInterview.getInterviewer()}</li>
           </ul>
+          <c:choose>
+            <c:when test= "${scheduledInterview.getHasPassed()}">
+              <a href="feedback.html?interview=${scheduledInterview.getId()}&role=${scheduledInterview.getRole()}" style="text-align:center">
+                <button class = "feedbackButton">Submit Feedback</button>
```

Comment:
Change the feedbackButton to a bootstrap style button to go along with the rest of the site

Reply:
Done